packages/shared/src/domain_model/ElectionSettings.ts

```diff
@@ -39,6 +39,7 @@ export interface ElectionSettings {
     contact_email?: string; // Public contact email for voters to reach out to
     exhaust_on_N_repeated_skipped_marks?: number; // number of skipped ranks before exhausting
     draggable_ballot?: boolean; // Use draggable interface for IRV ballots
+    redact_voter_ids?: boolean; // Redact voter IDs from all views except emergency break glass endpoint. Required for ballot_updates.
```

Comment:
Rather than having this as a separate setting, would it make sense to just have this be the behavior for all email-list based elections?